packages/excalidraw/components/LayerUI.scss

```
@@ -1,5 +1,6 @@
 @import "open-color/open-color";
 @import "../css/variables.module.scss";
+@import './emojiReactions/EmojiReactions.scss';
```

Stylelint (17.2.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)